src/Stack/Docker/GlobalDB.hs

```diff
@@ -76,11 +76,11 @@ getDockerImagesLastUsed config =
 pruneDockerImagesLastUsed :: Config -> [String] -> IO ()
 pruneDockerImagesLastUsed config existingHashes =
   withGlobalDB config go
-  where go = do l <- selectList [] []
-                forM_ l (\(Entity k (DockerImageProject{dockerImageProjectImageHash = h})) ->
-                           if h `elem` existingHashes
-                             then return ()
-                             else delete k)
+  where
+    go = do
+      l <- selectList [] []
+      forM_ l (\(Entity k DockerImageProject{dockerImageProjectImageHash = h}) ->
+                   unless (h `elem` existingHashes) $ delete k)
```

Comment:
How about `when (h `notElem` existingHashes)`?

I find `when` much clearer than `unless` in general, it reads way better.